wordpress-mysql-replication/azuredeploy.json

```diff
@@ -0,0 +1,378 @@
+            }
+        },
+        "webDeploy_SecureAuthenticationSalt": {
+            "type": "string",
+            "metadata": {
+                "description": "webdeploy secure authentication salt"
+            }
+        },
+        "webDeploy_LoggedInSalt": {
+            "type": "string",
+            "metadata": {
+                "description": "webdeploy logged in salt"
+            }
+        },
+        "webDeploy_NonceSalt": {
+            "type": "string",
+            "metadata": {
+                "description": "webdeploy nonce salt"
+            }
+        },
+        "dnsName": {
+            "type": "string",
+            "metadata": {
+                "description": "Connect to your cluster using dnsName.location.cloudapp.azure.com"
+            }
+        },
+        "wpDbUsername": {
+            "type": "string",
+            "defaultValue": "wpuser",
+            "metadata": {
+                "description": "wordpress db username"
+            }
+        },
+        "wpDbPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "wordpress db password"
+            }
+        },
+        "vmUserName": {
+            "type": "string",
+            "metadata": {
+                "description": "user name to ssh to the VMs"
+            }
+        },
+        "vmPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "password to ssh to the VMs"
+            }
+        },
+        "mysqlRootPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "mysql root user password"
+            }
+        },
+        "mysqlReplicationPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "mysql replication user password"
+            }
+        },
+        "mysqlProbePassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "mysql probe password"
+            }
+        },
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "Standard_A1",
+            "metadata": {
+                "description": "size for the VMs"
+            }
+        },
+        "storageAccountType": {
+            "type": "string",
+            "defaultValue": "Standard_LRS",
+            "metadata": {
+                "description": "Storage account type for the cluster"
+            }
+        },
+        "virtualNetworkName": {
+            "type": "string",
+            "defaultValue": "mysqlvnet",
+            "metadata": {
+                "description": "Virtual network name for the cluster"
+            }
+        },
+        "dbSubnetName": {
+            "type": "string",
+            "defaultValue": "dbsubnet",
+            "metadata": {
+                "description": "subnet name for the MySQL nodes"
+            }
+        },
+        "vnetAddressPrefix": {
```

Comment:
Comments about reducing to only the necessary parameters from above applies below as well